examples/Example/src/new_api/camera/index.tsx

```diff
@@ -0,0 +1,351 @@
+import React, { useState } from 'react';
+import { Alert, StyleSheet, useWindowDimensions } from 'react-native';
+import {
+  GestureMonitor,
+  Gesture,
+  useAnimatedGesture,
+} from 'react-native-gesture-handler';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  runOnJS,
+} from 'react-native-reanimated';
+
+const filters = ['red', 'green', 'blue', 'yellow', 'orange', 'cyan'];
+const MAX_VIDEO_DURATION = 60_000;
```

Comment:
```suggestion
const MAX_VIDEO_DURATION_MS = 60_000;
```

Reply:
Done in ae78c347855f8ebb233e8fa59d2e0794bc4691a3